src/index.html

```diff
@@ -9,6 +9,11 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+
+    <div class="watch">
+      <div class="watch__hand watch__hand--second"></div>
+      <div class="watch__hand watch__hand--minute"></div>
+    </div>
+
   </body>
```

Comment:
```suggestion
    <div class="watch">
      <div class="watch__hand watch__hand--second"></div>
      <div class="watch__hand watch__hand--minute"></div>
    </div>
  </body>
```
Don't leave empty lines between parent and child elements